.travis/runcoveralls.sh

```diff
@@ -0,0 +1,88 @@
+#!/bin/bash
+# SPDX-License-Identifier: MIT
+
+# Reports coverage results using coveralls. The aim of this script is to
+# provide a unified way to reporting coverage results across all linux system
+# roles projects.
+
+# The given command line arguments are passed to coveralls.
+
+# Environment variables:
+#
+#   LSR_PUBLISH_COVERAGE
+#     if the variable is empty or unset, nothing will be published; if the
```

Comment:
This help is missing the info about what to do to enable coveralls in non-strict and non-debug mode. Currently it seems to do this for any value but it would be better to do this for a certain value or set of values only.

Reply:
@tyll I don't know anything about `runcoveralls.sh` - I'm just trying to sync what @i386x added to the template with other repos - at this point I would rather drop it as it seems it is not fully baked and doesn't really work.